administrator/language/de-DE/com_admin.ini

```
@@ -68,18 +68,14 @@ COM_ADMIN_HELP_COMPONENTS_REDIRECT_MANAGER_EDIT="Umleitungen: Links - Neu/Bearbe
 COM_ADMIN_HELP_COMPONENTS_SEARCH="Suche"
 COM_ADMIN_HELP_COMPONENTS_TAGS_MANAGER="Schlagwörter (Tags)"
 COM_ADMIN_HELP_COMPONENTS_TAGS_MANAGER_EDIT="Schlagwörter (Tags): Neu/Bearbeiten"
-COM_ADMIN_HELP_COMPONENTS_WEBLINKS_CATEGORIES="Weblinks: Kategorien"
-COM_ADMIN_HELP_COMPONENTS_WEBLINKS_CATEGORIES_EDIT="Weblinks: Kategorien - Neu/Bearbeiten"
-COM_ADMIN_HELP_COMPONENTS_WEBLINKS_LINKS="Weblinks"
-COM_ADMIN_HELP_COMPONENTS_WEBLINKS_LINKS_EDIT="Weblinks: Neu/Bearbeiten"
 COM_ADMIN_HELP_CONTENT_ARTICLE_MANAGER="Beiträge"
 COM_ADMIN_HELP_CONTENT_ARTICLE_MANAGER_EDIT="Beiträge: Neu/Bearbeiten"
 COM_ADMIN_HELP_CONTENT_FEATURED_ARTICLES="Beiträge: Hauptbeiträge"
 COM_ADMIN_HELP_CONTENT_MEDIA_MANAGER="Medien"
 COM_ADMIN_HELP_EXTENSIONS_EXTENSION_MANAGER_DATABASE="Erweiterungen: Datenbank überprüfen"
 COM_ADMIN_HELP_EXTENSIONS_EXTENSION_MANAGER_DISCOVER="Erweiterungen: Überprüfen"
 COM_ADMIN_HELP_EXTENSIONS_EXTENSION_MANAGER_INSTALL="Erweiterungen: Installieren"
-COM_ADMIN_HELP_EXTENSIONS_EXTENSION_MANAGER_LANGUAGES="Erweiterungen: Sprachen installieren"
+COM_ADMIN_HELP_EXTENSIONS_EXTENSION_MANAGER_LANGUAGES="Erweiterungen: Akkreditierte Sprachen installieren"
```

hmm ist das nicht etwas lang?

---

ich würde es bei

```suggestion
COM_ADMIN_HELP_EXTENSIONS_EXTENSION_MANAGER_LANGUAGES="Erweiterungen: Sprachen installieren"
```
belassen